mmdet/models/losses/iou_loss.py

```diff
@@ -69,6 +69,60 @@ def bounded_iou_loss(pred, target, beta=0.2, eps=1e-3):
     return loss
 
 
+@weighted_loss
+def giou_loss(pred, target, eps=1e-6):
+    """GIoU loss.
+
+    https://github.com/sfzhang15/ATSS/blob/master/atss_core/modeling/rpn/atss/loss.py#L36
```

Comment:
Generalized Intersection over Union: A Metric and A Loss for Bounding Box Regression

https://arxiv.org/abs/1902.09630